src/main/java/duke/Event.java

```diff
@@ -0,0 +1,30 @@
+package duke;
+
+public class Event extends Task {
+    protected String type = "E";
+    protected String start;
+    protected String end;
+
+    /**
+     * Constructor for Event
+     * @param index index of the task
+     * @param description description of the task
+     * @param start start date and time of the task
+     * @param end end date and time of the task
+     */
+    public Event(int index, String description, String start, String end) {
+        super(index, description);
+        this.start = start;
+        this.end = end;
+    }
+
+    @Override
+    public String getTask() {
+        return index + ". [" + type + "][" + getStatusIcon() + "] " + description + "(from: " + start + " to: " + end + ")";
```

Comment:
Should your line length be longer than 120 chars?
```suggestion
        return index + ". [" + type + "][" + getStatusIcon() + "] "
                + description + "(from: " + start + " to: " + end + ")";
```